docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
             - PGDATA=/var/lib/postgresql/data/pgdata
             - POSTGRES_PASSWORD=pleaseChange # default postgres password that can be changed for security.
         volumes:
-            - ./postgres-data:/var/lib/postgresql/data/pgdata
+            - ./postgres-data:/var/lib/postgresql/pgdata
```

Comment:
Unsure why this was changed given the PGDATA defined above? Can you help me understand?

Reply:
mentioned this issue in #656 , in the meantime, reverted the change.